common/data/dataHelpers.ts

```diff
@@ -0,0 +1,12 @@
+﻿/* eslint-disable */
+export const StrapiBaseUrl = "https://map-api.ekaterinburg.io/api";
+
+export async function getDataJsonByUrl(url: string){
+    return await fetch(url)
+        .then(x => x.json());
```

Comment:
We should to use only `await` or only `then`.

await example:

const response = await fetch(url);
const data = await response.json();